news/2022-09-15-anniversary-art-contest-results.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: post
+title: "15th Anniversary Art Contest Results"
+date: 2022-09-15 10:30:00 +0000
+---
+
+After some of the largest voting turnout we've seen in a long time, the results of the 15th Anniversary Art Contest have made their way into a main menu near you!
+
+![](https://assets.ppy.sh/contests/153/banners/Dreamxiety.jpg)
+
+With **86** artists putting their best foot forward, it was never going to be an easy contest to decide, and when people asked us to increase the votes from 10 to 15, the future was all but set in stone.
+
+**14,930** voters made their preferences known across a wide variety of pieces exploring osu!'s (quite literally) colourful history, and now we have finalists to crown.
+
+We'll be getting stuck straight into the voting results below - but wait, there's a twist! The observant among you may have noticed that we've got far more than the standard 15-20 entries that usually make it in, and that's because we've got an extra *six* courtesy of peppy himself this time around.
```

Comment:
```suggestion
We'll be getting stuck straight into the voting results below — but wait, there's a twist! The observant among you may have noticed that we've got far more than the standard 15-20 entries that usually make it in, and that's because we've got an extra *six* courtesy of peppy himself this time around.
```